servers/v3-proxy-api/src/graph/graphQLClient.ts

```diff
@@ -47,22 +46,15 @@ export function getClient(
   if (accessToken && consumerKey) {
     url = `${config.graphQLProxy}?consumer_key=${consumerKey}&access_token=${accessToken}`;
   } else {
-    url = `${config.graphQLProxy}/?consumer_key=${consumerKey}`;
-  }
-
-  try {
-    return new GraphQLClient(url, {
-      headers: headers,
-      //fetch implementation used by node version,
-      //can give custom fetch package
-      fetch,
-    });
-  } catch (e) {
-    Sentry.addBreadcrumb({ message: `graphQLClient creation failed:` });
-    Sentry.captureException(e);
-    serverLogger.error(e);
-    throw e;
+    url = `${config.graphQLProxy}?consumer_key=${consumerKey}`;
   }
+  Sentry.addBreadcrumb({ message: `creating GraphQL client` });
+  return new GraphQLClient(url, {
+    headers: headers,
+    //fetch implementation used by node version,
+    //can give custom fetch package
+    fetch,
```

Comment:
ooo thats right node includes this now!

Reply:
can't take credit, this was just unindented I think haha